src/dataexplorer/dataexplorer-application/src/components/PatientInfo/PatientInfo.tsx

```diff
@@ -81,16 +80,11 @@ const PatientInfo = ({ patientID, setName, setSchema }: IPatientInfoProps) => {
     }, [questionnaire]);
 
     if (patientData && patientData !== undefined && patientData.total !== 0) {
-        const name =
-            patientData.entry[0].resource.name[0].given[0] +
-            ' ' +
-            patientData.entry[0].resource.name[0].family;
         // Since the search uses social security number, which are
         // unique, the response will contain a maximum value of 1,
         // if the patient exists in the database.
 
         if (patientData.total === 1) {
-            setName(name);
             return (
```

Comment:
Is it possible to just have one if sentence here?

Reply:
What do you mean, the comment?

Reply:
Yup. Done.